test/10-file.t

```diff
@@ -196,6 +196,10 @@ is $batch->[3]{task},       'fail',       'right task';
 is $batch->[3]{state},      'finished',   'right state';
 is $batch->[3]{retries},    0,            'job has not been retried';
 ok !$batch->[4], 'no more results';
+# Minion::Command::minion::job calls list_jobs this way
+$batch = $minion->backend->list_jobs(0, 10, {queue => undef, state => undef,
```

Comment:
e.g. modify https://github.com/kraih/minion/tree/master/examples/linkcheck to use Storable
```
# ./script/linkcheck minion job -e foo -a '[23, "bar"]'
# ./script/linkcheck minion job
Use of uninitialized value in string eq at .../lib/Minion/Backend/Storable.pm line 71.
```